pkgs/misc/vim-plugins/update.py

```diff
@@ -277,17 +285,17 @@ def check_results(
         sys.exit(1)
 
 
-def parse_plugin_line(line: str) -> Tuple[str, str, Optional[str]]:
+def parse_plugin_line(line: str) -> Tuple[str, str, str]:
     name, repo = line.split("/")
     try:
         repo, alias = repo.split(" as ")
         return (name, repo, alias.strip())
     except ValueError:
         # no alias defined
-        return (name, repo.strip(), None)
+        return (name, repo.strip(), "")
```

Comment:
Why did you change this?

Reply:
I believe it had something to do with mypy but it doesn't seem necessary any more. I'll try reverting it.